portfolio/src/main/java/com/google/sps/servlets/ListCommentsServlet.java

```diff
@@ -41,37 +46,33 @@
     private static final String COMMENT_PARAMETER = "comment";
     private static final String NAME_PARAMETER = "name";
     private static final String TIMESTAMP_PARAMETER = "timestamp";
+    private static final String MAX_COMMENTS_PARAMETER = "maxComments";
 
     @Override
     public void doGet (HttpServletRequest request, HttpServletResponse response) throws IOException {
         Query query = new Query("Comment").addSort(TIMESTAMP_PARAMETER, SortDirection.DESCENDING);
         DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
         PreparedQuery results = datastore.prepare(query);
-
         ArrayList<Comment> comments = new ArrayList<>();
-
-        
         // Streams entities to iterate through
-        try {
-            Stream<Entity> stream = StreamSupport.stream(results.asIterable().spliterator(), false);
-            stream.forEach(entity -> {
+        
+        Stream<Entity> stream = StreamSupport.stream(results.asIterable().spliterator(), false);
+        stream.forEach(entity -> {
+            if (!entity.equals(null)){
```

Comment:
Prefer entity !=null

It's harder to get a null pointer exception that way (which you can have happen if whoever implemented the equality check for entity.equals wrote it wrong